<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="btnBarClassificationScheme">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="btnLoadClasses">
       <property name="text">
        <string>Load</string>
       </property>
       <property name="icon">
        <iconset resource="eotsv_resources.qrc">
         <normaloff>:/eotimeseriesviewer/icons/copyright_label.svg</normaloff>:/eotimeseriesviewer/icons/copyright_label.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnRemoveClasses">
       <property name="text">
        <string>Load</string>
       </property>
       <property name="icon">
        <iconset resource="eotsv_resources.qrc">
         <normaloff>:/eotimeseriesviewer/icons/mActionRemove.svg</normaloff>:/eotimeseriesviewer/icons/mActionRemove.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnAddClasses">
       <property name="text">
        <string>Load</string>
       </property>
       <property name="icon">
        <iconset resource="eotsv_resources.qrc">
         <normaloff>:/eotimeseriesviewer/icons/mActionAdd.svg</normaloff>:/eotimeseriesviewer/icons/mActionAdd.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableClassificationScheme">
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="eotsv_resources.qrc"/>
 </resources>
 <connections/>
</ui>
